src/main/resources/static/index.html

```diff
@@ -1,126 +1,127 @@
-                alt="sign in">
-          </a>
-          <a href="#" class="head__login-buttons">
-            <img
-                src="data:image/svg+xml,%3Csvg width='25' height='25' viewBox='0 0 25 25' fill='none' xmlns='http://www.w3.org/2000/svg'%3E%3Cg clip-path='url(%23clip0)'%3E%3Cpath d='M12.3218 13.0426C13.9762 13.0426 15.4088 12.4492 16.5794 11.2785C17.7499 10.1079 18.3433 8.67571 18.3433 7.02109C18.3433 5.36705 17.7499 3.93463 16.5792 2.76372C15.4085 1.59337 13.976 1 12.3218 1C10.6672 1 9.23494 1.59337 8.0644 2.76391C6.89386 3.93444 6.30029 5.36686 6.30029 7.02109C6.30029 8.67571 6.89386 10.1081 8.06459 11.2787C9.23532 12.449 10.6677 13.0426 12.3218 13.0426Z' fill='%23F6A325'/%3E%3Cpath d='M22.8071 20.1478C22.7733 19.6606 22.705 19.1292 22.6045 18.5681C22.503 18.0028 22.3724 17.4683 22.216 16.9799C22.0544 16.475 21.8347 15.9764 21.5631 15.4986C21.2812 15.0027 20.9501 14.5709 20.5785 14.2155C20.19 13.8438 19.7143 13.5449 19.1642 13.3269C18.6161 13.1101 18.0086 13.0002 17.3587 13.0002C17.1035 13.0002 16.8567 13.1049 16.3801 13.4152C16.0867 13.6065 15.7436 13.8278 15.3606 14.0725C15.0331 14.2812 14.5895 14.4767 14.0415 14.6537C13.5068 14.8267 12.964 14.9144 12.4282 14.9144C11.8925 14.9144 11.3498 14.8267 10.8146 14.6537C10.2672 14.4769 9.82356 14.2814 9.49645 14.0727C9.11708 13.8303 8.77376 13.609 8.47602 13.415C7.99995 13.1047 7.75294 13 7.49774 13C6.84772 13 6.24042 13.1101 5.69244 13.3271C5.14274 13.5447 4.66685 13.8436 4.27795 14.2157C3.90659 14.5713 3.57528 15.0029 3.29375 15.4986C3.02234 15.9764 2.80261 16.4748 2.64087 16.98C2.48466 17.4685 2.354 18.0028 2.25253 18.5681C2.15202 19.1285 2.08373 19.6601 2.04997 20.1483C2.01678 20.6267 2 21.1232 2 21.6246C2 22.9296 2.41485 23.9861 3.23291 24.7652C4.04086 25.5341 5.10993 25.9241 6.40998 25.9241H18.4476C19.7477 25.9241 20.755 26.1496 21.5631 25.3806C22.3814 24.602 22.8576 22.93 22.8576 21.6244C22.8574 21.1207 22.8405 20.6238 22.8071 20.1478Z' fill='%23F6A325'/%3E%3C/g%3E%3Cdefs%3E%3CclipPath id='clip0'%3E%3Crect width='25' height='25' fill='white'/%3E%3C/clipPath%3E%3C/defs%3E%3C/svg%3E%0A"
-                alt="sign up">
-          </a>
-        </div>
-      </div>
-
-      <!-- header search -->
-
-      <div class="header__search">
+    <!-- header -->
+
+    <header class="header">
+        <div class="container">
+
+            <!-- header menu -->
+
+            <div class="header__row">
+                <nav class="header__menu" role="navigation">
+                    <ul class="header__menu-list">
+                        <li>
+                            <a href="#" class="header__menu-item">about</a>
+                        </li>
+                        <li>
+                            <a href="catalog" class="header__menu-item">catalog</a>
+                        </li>
+                        <li>
+                            <a href="index.html" class="header__logo">
+                                <img src="img/svg/wax_deals.svg" alt="wax deals logo">
```

Comment:
Don't do so please as i have set up 2 indents in webStorm. This makes it easier to view the html code

Reply:
Sorry, I'll change it.